### ISSUE TITLE
Suggestion~

### ISSUE THREAD

**Issue by lonelywm** [Kyubyong/deepvoice3#4]
I have read all of your codes but met some error, I then worked about one day to fix them, besides, I notice the bottleneck of the speed is data reading (the disk of my PC is not SSD, I have modified it), I also find that the mags for training may have something wrong, because I call spectrogram2wav(mag) to recover the origin wav, but what I got is just noise.(ho~ you have mod it)

Your network is very nice, I am looking forward to it, I wish you can get a good result soon.

**Comment by Kyubyong** [comment]
Thanks so much. Which parts did you change? I'm still changing/fixing my code. And did you apply denormalization before spectrogram2wav as in synthesize.py? Tomorrow I'll check what you said. Thanks!

**Comment by lonelywm** [comment]
Yes I do, I concat the origin mags to the generated. I will try again, maybe I have some mistakes.  Nevertheless, that is not the main problem, the network's architecture and hyperparams are, which always make me crazy.

**Comment by Kyubyong** [comment]
Thanks, let me double check on my end tomorrow.

**Comment by Kyubyong** [comment]
Oh you're right. When synthesizing, I should have set the base of log as 10, not exp. I've fixed it. Thanks.
